forte/data/data_store.py

```diff
@@ -184,9 +184,9 @@ def _new_tid(self) -> int:
         r"""This function generates a new ``tid`` for an entry."""
         return uuid.uuid4().int
 
-    def _new_annotation(self, type_name: str, begin: int, end: int) -> List:
-        r"""This function generates a new annotation with default fields. All
-        default fields are filled with None.
+    def _new_annotation(self, type_name: int, begin: int, end: int) -> List:
```

Comment:
May need to update the variable name (if it is an integer now), and the docstring.